scripts/integration.sh

```diff
@@ -13,148 +13,147 @@ npx hardhat compile --force
 echo "Testing omnichain Swap"
 
 OMNI_SWAP_CONTRACT=$(npx hardhat deploy --network zeta_testnet --json | jq -r '.address')
-# echo "omnichain.swap.zeta_testnet=$OMNI_SWAP_CONTRACT" >> contracts.toml
-OMNI_SWAP_TX=$(npx hardhat interact --contract $OMNI_SWAP_CONTRACT --network goerli_testnet --amount 0.01 --recipient $SENDER --destination mumbai_testnet --json | jq -r '.hash')
+OMNI_SWAP_TX=$(npx hardhat interact --contract $OMNI_SWAP_CONTRACT --network goerli_testnet --amount 0.01 --recipient $SENDER --target-token 0x48f80608B672DC30DC7e3dbBd0343c5F02C738Eb --json | jq -r '.hash')
 echo $OMNI_SWAP_TX
 OMNI_SWAP_CCTX=$(npx hardhat cctx $OMNI_SWAP_TX --json)
 echo $OMNI_SWAP_CCTX
 
 popd
 
-pushd "./omnichain/staking"
+# pushd "./omnichain/staking"
```

Comment:
it's intentional all the lines commented?

Reply:
This was an accidental change, I've reverted https://github.com/zeta-chain/example-contracts/pull/85/commits/b145cbf94f07e1e7b6e07334c88b5ba930af014b Thanks for catching it!